tests/snapshot.rs

```diff
@@ -35,7 +35,7 @@ struct Report {
     current_session_samples: Vec<usize>,
     current_silence_samples: Vec<usize>,
     current_speech_samples: Vec<usize>,
-    likelihoods: Vec<usize>,
+    likelihoods: Vec<f32>,
```

Comment:
You shouldn't be comparing floats for equality that's why it was made a usize. Maybe `prob.floor() as usize` would be a more sensible modification as it fixes the rounding direction to the same one for each architecture